pages/projects/[id]/index.js

```diff
@@ -0,0 +1,106 @@
+import Link from "next/link";
+import { useRouter } from "next/router";
+import Image from "next/image";
+import { FaArrowLeftLong } from "react-icons/fa6";
+import { uid } from "uid";
+import styled from "styled-components";
+
+export default function ProjectDetailsPage({ projects }) {
+  const router = useRouter();
+  const { id } = router.query;
+
+  const projectData = projects.find((project) => project.id === id);
+
+  if (!projectData) {
+    return null;

```

Comment:
```suggestion
    return <h1>No project found<h1>;

```
This is more suitable than a blank site